config.go

```diff
@@ -35,10 +36,22 @@ func defaultSocketPath() string {
 	return filepath.Join(home, ".ssh", "wsl2-ssh-agent.sock")
 }
 
+func commandPath(command string) string {
+	path := os.Getenv("PATH")
+	defer os.Setenv("PATH", path)
+	os.Setenv("PATH", path + ":/mnt/c/Windows/System32/WindowsPowerShell/v1.0")
```

Comment:
Rather than changing the `PATH` temporarily, I prefer to explicitly check if `/mnt/c/Windows/System32/WindowsPowerShell/v1.0/PowerShell.exe` is present when PATH discovery fails.